sources/web/datalab/static/datalab.css

```diff
@@ -13,7 +13,7 @@
  */
 
 /* Font Imports */
-@import url(https://fonts.googleapis.com/css?family=Source+Code+Pro:400,700|Open+Sans:400,700|Lato:400,700&subset=latin,latin-ext);
+@import url(https://fonts.googleapis.com/css?family=Source+Code+Pro:400,700|Open+Sans:400,700&subset=latin,latin-ext);
```

Comment:
Don't restrict Open Sans to just latin. We want to ensure non-latin characters don't show up as squares. Remove "|Lato:400,700&subset=latin,latin-ext"